### DISCUSSION TITLE
Configure Ticker with Alpaca Portfolio

### DISCUSSION THREAD

**Discussion by tbeseda** [achannarasappa/ticker#166] [Show and tell]
I made a quick Node CLI to populate Ticker's `.ticker.yaml` `lots` option with current positions from Alpaca.

[`alpaca-ticker-config` on npmjs.org](https://www.npmjs.com/package/alpaca-ticker-config) + documentation  
[source on github](https://github.com/tbeseda/alpaca-ticker-config)

I've listed some next steps under "Todo" in the README.

Probably a narrow set of users here, but it may be helpful for watching your Alpaca portfolio. 👍🏻

**Comment by achannarasappa** [comment]
This is awesome! It's great to see community built integrations for ticker. I'll add a section in the readme referencing this.
